src/components/Copyright/Copyright.js

```diff
@@ -38,8 +38,16 @@ class Copyright extends Component {
 
   updateCopyright() {
     const { layerService } = this.props;
+
+    const copyrights = layerService
+      .getLayersAsFlatArray()
+      .filter(l => l.getVisible() && l.getCopyright())
+      .map(l => l.getCopyright());
+
+    const unique = Array.from(new Set(copyrights));
```

Comment:
`copyrights` is already an array or I miss something

Reply:
It's for removing duplicates. I added a comment.